main/src/com/google/refine/expr/functions/strings/LevenshteinDistance.java

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
Please include our standard module header.

Reply:
If we care about this I'd rather add automation to take care of it rather than asking contributors to do it manually…

Reply:
I'm fine with automation. Most IDEs can be set up to use a standard module template. As a matter of fact, we used to have this in the Eclipse project settings. We could restore that or add external tooling (or even change the code style if that's what the team wants).

Reply:
There is the `mvn license:update-file-header` command which could be run from time to time (with the plugin appropriately configured), or integrated in `./refine lint`.
Also maybe the shorter license header could be used:
```
//  SPDX-FileCopyrightText: OpenRefine contributors
//  SPDX-License-Identifier: BSD-3-Clause
```

Reply:
I do like the shorter license header.